python/dgl/dataloading/dataloader.py

```diff
@@ -66,7 +67,7 @@ def __next__(self):
         batch = self._next_indices()
         if self.mapping_keys is None:
             # clone() fixes #3755, probably.  Not sure why.  Need to take a look afterwards.
-            return batch.clone()
+            return batch
```

Comment:
Did you find out the root cause of L69?

Reply:
Reverted.  This still works on some machines while not work on others...